tests/multinode.rs

```diff
@@ -956,6 +956,7 @@ fn test_leader_to_validator_transition() {
     info!("Check the ledger to make sure it's the right height...");
     let bank_forks = new_banks_from_blocktree(
         &leader_ledger_path,
```

Comment:
Any reason we don't put this someplace relative to the ledger files?

Reply:
You might want the accounts and the ledger on different hardware.

Reply:
We don't have that requirement today.

Reply:
Why not? We are already experimenting with putting the accounts on multiple NVME drives which are dedicated to the accounts storage.

Reply:
Good point. Already, that works.